packages/frameworks-sveltekit/src/lib/index.ts

```diff
@@ -215,6 +215,14 @@ export type {
   User,
 } from "@auth/core/types"
 
+export type {
+  Adapter,
+  AdapterAccount,
+  AdapterSession,
+  AdapterUser,
+  VerificationToken,
+} from "@auth/core/adapters"
+
```

Comment:
To keep it consistent, maybe add `adapter.ts`?

Needs to be updated in package.json#exports too.

(you can verify via `pnpm pack`)

Reply:
Yeah like copying over the file liek with providers? I was thinking about it, but we dont need all the additional stuff, right - its really just these few types :thinking:

Reply:
No, create an `adapter.ts` file and put

```ts
export type {
  Adapter,
  AdapterAccount,
  AdapterSession,
  AdapterUser,
  VerificationToken,
} from "@auth/core/adapters"
```

in it.